RiotSwiftUI/Modules/UserSessions/UserSessionCardView/UserSessionCardView.swift

```diff
@@ -0,0 +1,168 @@
+//
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+import DesignKit
+
+struct UserSessionCardView: View {
+    
+    @Environment(\.theme) var theme: ThemeSwiftUI
+    
+    var viewData: UserSessionCardViewData
+    
+    var onVerifyAction: ((String) -> (Void))? = nil
+    var onViewDetailsAction: ((String) -> (Void))? = nil
+    var onLearnMoreAction: (() -> (Void))? = nil
+    
+    private var verificationStatusImageName: String {
+        return viewData.isVerified ? Asset.Images.userSessionVerified.name : Asset.Images.userSessionUnverified.name
+    }
+    
+    private var verificationStatusText: String {
+        return viewData.isVerified ? VectorL10n.userSessionVerified : VectorL10n.userSessionUnverified
+    }
+    
+    private var verificationStatusColor: Color {
+        return viewData.isVerified ? theme.colors.accent : theme.colors.alert
+    }
+    
+    private var verificationStatusAdditionalInfoText: String {
+        return viewData.isVerified ? VectorL10n.userSessionVerifiedAdditionalInfo : VectorL10n.userSessionUnverifiedAdditionalInfo
+    }
+    
+    private var backgroundShape: RoundedRectangle {
+        return RoundedRectangle(cornerRadius: 8)
+    }
+    
+    private var showExtraInformations: Bool {
+        return viewData.isCurrentSessionDisplayMode == false && (viewData.lastActivityDateString.isEmptyOrNil == false || viewData.lastSeenIPInfo.isEmptyOrNil == false)
+    }
+    
+    var body: some View {
+        VStack() {
```

Comment:
This outer `VStack` doesn’t look necessary to me from a static review as there’s only one item in it which is another `VStack`